src/test/java/reposense/parser/ArgsParserTest.java

```diff
@@ -207,11 +207,6 @@ public void parse_configOrLocationsSimilar_success() throws ParseException, IOEx
         Assert.assertEquals(actualRepoConfigs, expectedRepoConfigs);
     }
 
-    @Test(expected = ParseException.class)
-    public void emptyArgs_throwsParseException() throws ParseException {
```

Comment:
should add a new test to check that `emptyArgs` works properly, also should return a `ConfigCliArguments` that has cwd set as the `configFolderPath`